examples/gettingStarted/scheduledMessages.ts

```diff
@@ -0,0 +1,104 @@
+/*
+  This sample demonstrates how .scheduleMessage() function can be used to schedule messages to appear
+  on a Service Bus entity at a specified later time.
+*/
+
+import {
+  Namespace,
+  SendableMessageInfo,
+  OnMessage,
+  MessagingError,
+  OnError,
+  ServiceBusMessage
+} from "../../lib";
+import { delay } from "rhea-promise";
+
+// Define connection string and related Service Bus entity names here
+const connectionString = "";
+const queueName = "";
+
+let ns: Namespace;
+
+const listOfScientists = [
+  { name: "Einstein", firstName: "Albert" },
+  { name: "Heisenberg", firstName: "Werner" },
+  { name: "Curie", firstName: "Marie" },
+  { name: "Hawking", firstName: "Steven" },
+  { name: "Newton", firstName: "Isaac" },
+  { name: "Bohr", firstName: "Niels" },
+  { name: "Faraday", firstName: "Michael" },
+  { name: "Galilei", firstName: "Galileo" },
+  { name: "Kepler", firstName: "Johannes" },
+  { name: "Kopernikus", firstName: "Nikolaus" }
+];
+
+async function main(): Promise<void> {
+  ns = Namespace.createFromConnectionString(connectionString);
+  try {
+    await sendScheduledMessages();
+
+    await receiveMessages();
+  } finally {
+    await ns.close();
+  }
+}
+
+// Scheduling messages to be sent after 30 seconds from now
+async function sendScheduledMessages(): Promise<void> {
+  // If using Topics, use createTopicClient to send to a topic
+  const client = ns.createQueueClient(queueName);
+
+  const messages: SendableMessageInfo[] = listOfScientists.map((scientist) => ({
+    body: `${scientist.firstName} ${scientist.name}`,
+    label: "Scientist"
+  }));
+
+  const timeNowUtc = new Date(Date.now());
+  const scheduledEnqueueTimeUtc = new Date(Date.now() + 10000);
+  console.log(`>>>> Time now in UTC: ${timeNowUtc}`);
+  console.log(`>>>> Messages will appear in Service Bus at UTC: ${scheduledEnqueueTimeUtc}`);
+
+  await client.scheduleMessages(scheduledEnqueueTimeUtc, messages);
+  for (let index = 0; index < messages.length; index++) {
+    await client.scheduleMessage(scheduledEnqueueTimeUtc, messages[index]);
+    console.log(`Sent: ${messages[index].body} - ${messages[index].label}`);
+  }
```

Comment:
Is the intention here to demonstrate both `scheduleMessage` and `scheduleMessages`?
I would prefer to keep the sample simpler, and just do one of the 2.